scylla/templates/scylla-statefulset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name:  {{ template "scylla.fullname" . }}
+  labels:
+    app: {{ template "scylla.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: {{ template "scylla.fullname" . }}
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "scylla.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "scylla.fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "scylla.fullname" . }}
+        image:  "{{ .Values.Image.repository }}:{{ .Values.Image.tag }}"
+        imagePullPolicy: {{ default "" .Values.Image.pullPolicy | quote }}
+        ports:
+          - containerPort: {{ .Values.Statefulset.intraPort }}
+            name: {{ .Values.Statefulset.intraName }}
+          - containerPort: {{ .Values.Statefulset.tlsIntraPort }}
+            name: {{ .Values.Statefulset.tlsIntraName }}
+          - containerPort: {{ .Values.Statefulset.jmxPort }}
+            name: {{ .Values.Statefulset.jmxName }}
+          - containerPort: {{ .Values.Service.port }}
```

Comment:
use here a custom port for `cql` (whatever that is) and add a `targetPort` property in the `service`. It's clearer.